index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta name="color-scheme" content="light dark" />
+    <meta name="color-scheme" content="light" />
```

Comment:
Why this change?

TD or Scripting has it as well, see https://github.com/w3c/wot-thing-description/blob/4b6de63bad06a4307b6f2683bc4e2ed3ad0d0802/index.html#L7

Reply:
I see